File: app/src/main/java/com/v2/desafionubank/model/ReasonDetails.java
package com.v2.desafionubank.model;

/**
 * Created by CaioSChristino on 15/04/18.
 */

public class ReasonDetails {
    private String id;

    private String title;

    private boolean selected;

    public String getId ()
    {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public void setSelected(boolean selected) {
        this.selected = selected;
    }

    public boolean isSelected() {
        return selected;
    }
}

File: app/src/main/java/com/v2/desafionubank/model/ReasonDetails.kt
package com.v2.desafionubank.model

/**
 * Created by CaioSChristino on 18/04/18.
 */


class ReasonDetails constructor(private val _id: String? = null,
                                private val _title: String? = null,
                                private val _isSelected: Boolean? = null) {
    val id: String? = _id
    val title: String? = _title
    var isSelected: Boolean? = _isSelected
}
